docs/mobility/motion/algorithms.md

```diff
@@ -31,7 +35,7 @@ This is to ensure that paths will be found when using defaults.
 CBiRRT returns the first valid path that it finds.
 The CBiRRT algorithm Viam uses is based on the algorithm described in the paper [Manipulation Planning on Constraint Manifolds](https://www.ri.cmu.edu/pub_files/2009/5/berenson_dmitry_2009_2.pdf)
 
-## How the motion service applies these algorithms
+### How the motion service applies these algorithms
```

Comment:
@biotinker is this section still accurate?  Should it be improved?

Reply:
This is still accurate. We haven't changed arm stuff.